docs/contract_types.rst

```diff
@@ -236,14 +236,20 @@ Options used by multiple contracts
   contract be kept instead.
 
   Wildcards (in the form of ``*``) are supported. These can stand in for a module names, but they do not extend to
-  subpackages.
+  subpackages. A recursive wildcard (in the form of ``**``) is also supported.
 
   Examples:
 
   - ``mypackage.*``:  matches ``mypackage.foo`` but not ``mypackage.foo.bar``.
   - ``mypackage.*.baz``: matches ``mypackage.foo.baz`` but not ``mypackage.foo.bar.baz``.
   - ``mypackage.*.*``: matches ``mypackage.foo.bar`` and ``mypackage.foobar.baz``.
   - ``mypackage.foo*``: not a valid expression. (The wildcard must replace a whole module name.)
+  - ``mypackage.**``: matches ``mypackage.foo.bar`` and ``mypackage.foobar.baz``.
```

Comment:
Should this be `mypackage.foo.bar.baz`?

Reply:
Well, no, I guess it was highlighting that it'd give the same as `mypackage.*.*`, but I guess also anything below that too. A bit of a tricky one to express sanely.